haas/ext/switches/brocade.py

```diff
@@ -60,33 +60,41 @@ def session(self):
     def disconnect(self):
         pass
 
-    def apply_networking(self, action):
+    def modify_port(self, port, channel, network_id):
         """ Apply a NetworkingAction to the switch.
```

Comment:
I'm going to just kill the docstring, since it's redundant with the one for th superclass. Thanks for spotting it.